### ISSUE TITLE
Multiple selected categories of tokens are not being exported

### ISSUE THREAD

**Issue by A143447** [six7/figma-tokens#189]
We have several categories of tokens that build on each other: core, bridging and components. (Components referring to bridging and bridging referring to core). 
In the Figma Tokens plugin, JSON tab, when all the categories are selected and export is clicked, only the 'highlighted' category tokens are in the export file. This is preventing the tokens from being resolved to the lowest value. Can you please assist?

![image](https://user-images.githubusercontent.com/39042077/128132319-1333092c-530c-458d-838c-458d4b245944.png)

**Comment by six7** [comment]
I agree, this would be great.

Would you expect that the resulting export be
a) Merged, so that if multiple themes were selected and multiple tokens share the same name the "latter" wins and only one makes it into the export?
b) Merged by theme, so that you'd get something like this:
```
Core tokens: {
  ...
},
Bridging Tokens: {
  ...
}
```
c)
Get several files, one per theme
d)
All of the above, user-customizable in dialog

**Comment by A143447** [comment]
Thanks for your response. From the options above, I prefer a, in our set up we have 'dark' tokens that are used for dark mode, it would be great if the 'latter' one would win when combining all the core (holds light mode colours) and dark tokens.

**Comment by six7** [comment]
A fix for this was introduced recently. The export tab now shows an option to include all token sets! You now get one file with the various token sets included. We'll expand on that in the future to include e.g. an export per file or a merged export. But for now this allows the most flexibility.

![image](https://user-images.githubusercontent.com/4548309/150674034-639d770a-77e4-48c5-b571-b4f7c779501d.png)

**Comment by A143447** [comment]
Hey @six7, Thank you so much for working on my request above, all of the token sets are now able to be exported (as per your message above). I did make an assumption when I raised the request that if all of the sets were exported, the token name would no longer be displayed in the 'value' field but rather the resolved value, so as per the example above, I expected the value to be 20px instead of $spacing.bridging.small. As this is an extension of the now closed issue, I will raise a new issue with this request but linking to the original (this) closed issue. Thanks again
